test/number/sign.wren

```diff
@@ -0,0 +1,6 @@
+IO.print(123.sign)      // expect: 1
+IO.print((-123).sign)   // expect: -1
+IO.print(0.sign)        // expect: 0
+IO.print((-0).sign)     // expect: 0
```

Comment:
According to the definition of sign-function, yes that's what I expect to return. In fact, `0` and `-0` are both, ehm, zero... hence their sign is zero.

What's your opinion?